javascripts/search.js

```diff
@@ -608,6 +608,15 @@ function searchCtrlFunc($scope, $window, searchService) {
       pages: pages,
       lastVisible: lastVisible
     };
+
+    if (isSearch && $scope.paginate.currentPage > 1) {
```

Comment:
Only problem I see with this is that it will never scroll to top if they go back to the first page. Seems like it might be better to just put a click event on any of the page buttons to scroll to the top.